app/pages/project/index.jsx

```diff
@@ -86,23 +88,17 @@ class ProjectPageController extends React.Component {
       }
     }
   }
+
   componentWillUnmount() {
     Split.clear();
   }
 
-  setupSplits(props) {
-    if (!props) {
-      ({ props } = this);
-    }
-    const { user } = props;
-    const { owner, name } = props.params;
-
+  setupSplits(slug, user) {
```

Comment:
This returns a promise to return a splits object. Can it be renamed to `getSplits` or awaitSplits`?